frontend/wallet/src/render/views/settings/CompressorSettings.re

```diff
@@ -0,0 +1,158 @@
+open Tc;
+
+module Styles = {
+  open Css;
+
+  let label =
+    merge([
+      Theme.Text.Body.semiBold,
+      style([color(Theme.Colors.midnight), marginBottom(`rem(0.25))]),
+    ]);
+};
+
+type modalState =
+  | Delegate(option(PublicKey.t), option(Int64.t))
+  | Stake(option(Int64.t));
+
+let intForState = state =>
+  switch (state) {
+  | Delegate(_, _) => 0
+  | Stake(_) => 1
+  };
+
+module CompressorModal = {
+  [@react.component]
+  let make = (~state, ~updateModal) =>
+    <Modal
```

Comment:
I prefer these like:
```
<Modal
  title="Change consensus"
  onRequestClose={_ => updateModal(_ => None)}
>
```

if im alone on this, then ignore!

Reply:
reformat decides

Reply:
RIP